plugins/GSdx/GSTextureCache.cpp

```diff
@@ -470,27 +472,14 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 		{
 			dst->m_texture->SetScale(GSVector2((float)multiplier, (float)multiplier));
 		}
-		else  // Custom resolution hack
+		else
 		{
-			GSVector4i fr = m_renderer->GetFrameRect();
-
-			int ww = (int)(fr.left + m_renderer->GetDisplayRect().width());
-			int hh = (int)(fr.top + m_renderer->GetDisplayRect().height());
-
-			// Gregory: I'm sure this sillyness is related to the usage of a 32bits
-			// buffer as a 16 bits format. In this case the height of the buffer is
-			// multiplyed by 2 (Hence a scissor bigger than the RT)
-
-			// This vp2 fix doesn't work most of the time
-
-			if(hh < 512 && m_renderer->m_context->SCISSOR.SCAY1 == 511) // vp2
-			{
-				hh = 512;
-			}
+			int width = int(m_renderer->GetDisplayRect().width());
+			int height = int(m_renderer->GetDisplayRect().height());
 
-			if(ww && hh)
+			if(width && height)
 			{
-				dst->m_texture->SetScale(GSVector2((float)w / ww, (float)h / hh));
+				dst->m_texture->SetScale(GSVector2((float)(m_custom_width) / width, (float)(m_custom_height) / height));
 			}
```

Comment:
Is there a reason you're re-declaring the custom_width/height again withing GSTextureCache?. Instead of calling eg: GSVector2i fb = m_renderer->GetInternalResolution();

The using dst->m_texture->SetScale(GSVector2((float)(fb.x) / width, (float)(fb.y) / height));